### PR TITLE
initial work on a port to BfW 1.16

### LINKED ISSUE #34: Add indicator to show which units are wearing snowshoes
This is basically a dup of cooljeanius/Son_Of_The_Black_Eye_Easy_Edition#15 except for ANO; IOW, a reminder to myself to port the snowshoe-related portion of cooljeanius/Son_Of_The_Black_Eye_Easy_Edition@74c91d3 over to this campaign.

### PR DESCRIPTION
**Pull request by cooljeanius**
I've started porting this campaign to BfW 1.16 and figured I'd open a PR for it at this point. It's not ready to publish on the add-on server yet, as all I've done to test it so far is to try loading it and debug my way thru the first scenario, but I figured I'd send a PR now rather than later, so the final PR doesn't end up as huge as my previous one... Anyways, this addresses #35, ought to close #34, does a little bit for bug #28, and addresses checkbox 2 of bug #22 a bit.

### CODE REVIEW

File: scenarios/19c_The_Oracle.cfg [hunk 1]
```diff
@@ -1490,7 +1488,13 @@
                 condition=lose
             [/objective]
             {TURNS_RUN_OUT}
+#ifdef EASY
+            #po: EASY difficulty, so be more explicit about how things work:
+            {OBJECTIVE_NOTES _"The scenario will continue even if Matthias Ramon dies; it is the keep upon which he stands that requires greater protection."}
```
Comment (nemaara) [inline scenarios/19c_The_Oracle.cfg:1493]: I don't get the point of this. It's just duplicating information in the other objectives, which literally tell you that you lose if an enemy unit reaches the keep.

Comment (cooljeanius) [reply scenarios/19c_The_Oracle.cfg:1493]: Recency bias; a player reading the part about it being ok to let Matthias Ramon dies might then think that it's ok to stop trying to protect him; this reminds them that that's not the case. It helps players realize the distinction that prevents "don't let them reach the keep" and "Matthias Ramon can die" from being conflicting points of information. It's the same reason we have introductions and conclusions in regular writing that just restate the actual body of the text, to drive home the most important parts, and help the reader synthesize.

Comment (nemaara) [reply scenarios/19c_The_Oracle.cfg:1493]: Yeah I think it's not a good idea. Objective clarity should be a thing for all modes, not just easy mode. It's one thing to give extra strategic hints for easy, but this is not a hint. If you think the objectives are not clear enough then you can just make it for all modes.

Comment (nemaara) [reply scenarios/19c_The_Oracle.cfg:1493]: Also you'll need to reword it to get rid of the semi colon.

Comment (cooljeanius) [reply scenarios/19c_The_Oracle.cfg:1493]: > Yeah I think it's not a good idea. Objective clarity should be a thing for all modes, not just easy mode. It's one thing to give extra strategic hints for easy, but this is not a hint. If you think the objectives are not clear enough then you can just make it for all modes.

My thinking for just doing it on easy is that people who play on harder difficulties are probably better at paying attention than the scatterbrains like me who play on easy, but if you think people who play on harder difficulties could use it, too, then I guess I can put it back to adding the clarification unconditionally.

> Also you'll need to reword it to get rid of the semi colon.

What's wrong with the semicolon? It's grammatically correct; it links two independent clauses that could stand on their own as independent sentences, but that make more sense when joined together.

Comment (nemaara) [reply scenarios/19c_The_Oracle.cfg:1493]: It's an older style. Modern editing is moving away from using semicolons.

Comment (cooljeanius) [reply scenarios/19c_The_Oracle.cfg:1493]: Well I prefer the older style.

Comment (cooljeanius) [reply scenarios/19c_The_Oracle.cfg:1493]: If you can think of an alternate wording that avoids the semicolon, I'd be amenable to you using that, but if I'm the one who's supposed to be rewording the sentence, I'd rather just keep the semicolon.

**Comment by cooljeanius** [comment]
cc @knyghtmare

File: scenarios/19c_The_Oracle.cfg (hunk 1 above)
Comment (nemaara) [reply scenarios/19c_The_Oracle.cfg:1493]: Probably better to just have it be a period separating the two sentences.

Comment (cooljeanius) [reply scenarios/19c_The_Oracle.cfg]: > Probably better to just have it be a period separating the two sentences.

Ok, changed in 6e1cdc8
